invokeai/backend/model_manager/load/model_loaders/flux.py

```diff
@@ -84,7 +84,15 @@ def _load_model(
             model = AutoEncoder(ae_params[config.config_path])
             sd = load_file(model_path)
             model.load_state_dict(sd, assign=True)
-            model.to(dtype=self._torch_dtype)
+            # VAE is broken in float16, which mps defaults too
+            if self._torch_dtype == torch.float16:
+                try:
+                    vae_dtype = torch.tensor([1.0], dtype=torch.bfloat16, device=self._torch_device).dtype
+                except TypeError:
+                    vae_dtype = torch.tensor([1.0], dtype=torch.float32, device=self._torch_device).dtype
```

Comment:
No need to test `torch.float32`. It should always work, and even if it didn't we have no further fallback.
```suggestion
                    vae_dtype = torch.float32
```